apps/docs/pages/guides/platform/custom-domains.mdx

```diff
@@ -18,139 +18,213 @@ Custom domains allow you to present a branded experience to your users. Custom d
+4. Get a custom domain. Currently, only subdomains are supported.
+   - Use `api.example.com` instead of `example.com`.
+
+### Add a CNAME record
 
-### Configure a CNAME
+To make sure your custom domain points to the Supabase project you need to add a CNAME record to your domain's DNS settings.
 
-Set up a CNAME record for `api.example.com`, resolving to `foobarbaz.supabase.co.`, with as low a TTL as possible.
+If your project's default domain is `abcdefghijklmnopqrst.supabase.co` you should:
 
-### Configure TXT Verification
+- Create a CNAME record for `api.example.com` that resolves to `abcdefghijklmnopqrst.supabase.co.`.
+- Use a low TTL value so that you can see changes propagating quickly if you make a mistake.
 
-Use the `create` subcommand of the CLI to notify Supabase of your domain and retrieve TXT verification records:
+### Verify ownership of the domain
+
+In order for Supabase to know that you're the rightful owner of the domain, you need to add special TXT records that prove your ownership of it.
+
+Use the [`domains create`](/docs/reference/cli/supabase-domains-create) command of the CLI to register the domain with Supabase and retrieve TXT verification records:
 
 ```bash
-supabase domains create --project-ref foobarbaz --custom-hostname api.example.com --experimental
+supabase domains create --project-ref abcdefghijklmnopqrst --custom-hostname api.example.com
 ```
 
-The output of the `create` command includes two TXT records[^1] you will need to set up, in order to verify your control over the domain in question,
-and for us to issue SSL certificates for it. For example:
+Two TXT records are returned and you will need to add them to the domain's DNS settings. For example:
 
-```bash
+```text
 [...]
 Required outstanding validation records:
         _cf-custom-hostname.api.example.com TXT -> 46BBC14D-D50A-409C-8DB5-F862CF5BA660
         api.example.com TXT -> ca3-F1HvR9i938OgVwpCFwi1jTsbhe1hvT0Ic3efPY3Q
 ```
 
-[^1] One of the records requires you to _replace_ the CNAME record set up in [the first step](#configure-a-cname) with a TXT record. You'll be able to restore it back to the CNAME after the verification process has been completed.
+When adding the TXT records make sure you remove any extra spaces surrounding the values. Use a low TTL value so you can quickly change them if you make a mistake.
 
 ### Verify your domain
 
-Set up both records as instructed, and then use the `reverify` command for the Supabase Platform to verify the records:
+Once you have setup the DNS records:
+
+- CNAME for your custom domain pointing to the Supabase project domain.
+- TXT record for `_cf-custom-hostname.<domain>`.
+- TXT record for `<domain>`.
+
+Use the [`domains reverify`](/docs/reference/cli/supabase-domains-reverify) command to begin the verification process of your domain. You may need to run this command a few times as DNS records take a while to propagate.
```

Comment:
```suggestion
Use the [`domains reverify`](/docs/reference/cli/supabase-domains-reverify) command to begin the verification process of your domain. You may need to run this command a few times because DNS records take a while to propagate.
```